thunder/tests/test_jit_general.py

```diff
@@ -613,7 +613,7 @@ def test_nanogpt():
         "falcon-7b-like",
         "falcon-40b-like",
         "codellama2-like",
-        pytest.param("mixtral-like", marks=pytest.mark.xfail(raises=NotImplementedError, reason="topk", strict=True)),
+        pytest.param("mixtral-like", marks=pytest.mark.xfail(raises=TypeError, reason="topk", strict=True)),
```

Comment:
What is the error now?

Reply:
It fails with missing args for where. A yet another folloup

Reply:
@nikitaved Can you fill this in a new issue?